index.js

```diff
@@ -1,9 +1,9 @@
 var express = require("express");
-var http = require("http");
 var app = express();
+var dao = require('./app/dao/dao');
+var port = process.env.PORT || 8000;
```

Comment:
what is process.env.PORT? we dont have jenkins

Reply:
Heroku adds the allowed port into this env variable.